src/main/java/org/launchcode/liftoffproject/models/Reward.java

```diff
@@ -83,4 +94,7 @@ public LocalDate getRedemptionDate() {
     public void setRedemptionDate(LocalDate redemptionDate) {
         this.redemptionDate = redemptionDate;
     }
+
+    public void setReward(Reward redeemedReward) {
+    }
```

Comment:
```suggestion
```
I'm not sure we need this method here—I can't quite parse what it's supposed to do.